src/Controller/Action/DownloadGiftCardPdfAction.php

```diff
@@ -25,18 +19,18 @@ final class DownloadGiftCardPdfAction
     /** @var FlashBagInterface */
     private $flashBag;
 
-    public function __invoke(Request $request, int $giftCardId): Response
-    {
-        $giftCard = $this->giftCardRepository->find($giftCardId);
-        if (!$giftCard instanceof GiftCardInterface) {
-            throw new NotFoundHttpException('Gift card not found');
-        }
-        if (!$this->authChecker->isGranted(GiftCardVoter::READ, $giftCard)) {
-            $this->flashBag->add('error', 'setono_sylius_gift_card.gift_card.read_error');
-
-            return new RedirectResponse(filter_var($request->headers->get('referer'), \FILTER_SANITIZE_URL));
-        }
-
-        $configuration = $giftCard->getConfiguration();
-    }
+//    public function __invoke(Request $request, int $giftCardId): Response
```

Comment:
Please, do not commit commented code blocks (even in intermediate commits)

Reply:
Ok